components/server/src/ome/security/SystemTypes.java

```diff
@@ -78,6 +79,8 @@ public boolean isSystemType(Class<?> klass) {
             return true;
         } else if (IGlobal.class.isAssignableFrom(klass)) {
             return true;
+        } else if (Namespace.class.isAssignableFrom(klass)) {
```

Comment:
Unless there's a reason not to from @jburel, @manics, etc., I'd go for global and we do whatever cleanup of those tables & views that's necessary/appropriate.